aptos-move/block-executor/src/view.rs

```diff
@@ -353,7 +425,48 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> TAggregator
         state_key: &Self::IdentifierV1,
         _mode: AggregatorReadMode,
     ) -> anyhow::Result<Option<StateValue>> {
-        // TODO: Integrate aggregators.
+        // TODO: Integrate aggregators V1.
```

Comment:
I don't understand this TODO from before, nor the updated one. :) What is exactly needed here? doesn't this work correctly for V1?

Reply:
We can lift aggregator value using MoveTypeLayout::U128, so we redirect to V2 flow, and the V1 state item is treated as data.

Reply:
Made TODO more clear